clients/include/utility.hpp

```diff
@@ -19,7 +19,18 @@
  * \brief provide common utilities
  */
 
-// Passed into gtest's SUCCEED macro when skipping a test.
+// We use std::cout and std::cerr instead of stdout, stderr, std::cout and std::cerr
+// This must come after the header #includes above, to avoid poisoning system headers.
```

Comment:
should this be:   We use rocblas_cout and rocblas_cerr instead of stdout, stderr, std::cout and std::cerr

Reply:
That comment is, again, part of the mechanical changes. I basically undid the mechanical changes in the other PR to come up with this PR, so it replaced rocblas_cout and rocblas_cerr with std::cout and std::cerr, even in the comments. Refer to PR#922 for the real code if you have questions about this code and the differences look mechanical like they were edited by a sed script.